Services/Repository.cs

```diff
@@ -15,47 +19,46 @@ public interface IRepository
 
 public class Repository : IRepository 
 {
-    public static Dictionary<Guid, ToDo> listToDo = new Dictionary<Guid, ToDo>() {};
+    private readonly ApplicationContext _context;
+    public DbSet<ToDo> _dbSet => _context.Set<ToDo>();
+
+    public Repository( ApplicationContext context)
+    {
+        _context = context;
+    }
 
     public ToDo[] GetAll() 
     {
-        return listToDo.Values.ToArray();
+        return _dbSet.ToArray();
```

Comment:
Done. Question: Is it designating it as an array that makes too specific? Like you'd want this to work with any enumerable data structure?

Reply:
Yes, you have to decide upon what the calling code's need is, in this case the calling code only needs to be able to loop over the data structure because it needs to translate it into json

therefore `IEnumerable<T>` is the most appropriate type.